libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/saveAsDashboardHandler.ts

```diff
@@ -132,28 +134,36 @@ export function* saveAsDashboardHandler(
     ctx: DashboardContext,
     cmd: SaveDashboardAs,
 ): SagaIterator<DashboardCopySaved> {
-    const saveAsCtx: SagaReturnType<typeof createDashboardSaveAsContext> = yield call(
-        createDashboardSaveAsContext,
-        cmd,
-    );
-    const {
-        payload: { switchToCopy },
-    } = cmd;
-    const result: SagaReturnType<typeof saveAs> = yield call(saveAs, ctx, saveAsCtx);
-    const { dashboard, batch } = result;
-
-    if (batch) {
-        yield put(batch);
+    try {
+        yield put(dashboardCommandStarted(ctx, cmd));
```

Comment:
I think this is better in the rootCommandHandler saga. let's emit these events for all commands right away, then plan for a task to clean up the duplicate *Requested events.

Reply:
Good point, fixed in 0652b73